StringCalculator.tests/StringCalculatorTests.cs

```diff
@@ -0,0 +1,111 @@
+﻿using NUnit.Framework;
+
+namespace StringCalculator.tests;
+public class StringCalculatorTests
+{
+    private StringCalculator _calculator = null!;
+
+    [SetUp]
+    public void Setup()
+    {
+        _calculator = new StringCalculator();
+    }
```

Comment:
Nice find on the setup method. Do you know why this is better than some alternatives (like using the constructor)?